.github/workflows/ci-arm-build.yml

```diff
@@ -0,0 +1,67 @@
+name: CI ARM64 Build and Push
+
+on:
+  # push:
+  #   branches:
+  #     - "master"
+  #   tags-ignore:
+  #     - "*"
+  # pull_request:
+  #   branches-ignore:
+  #     - "*"
+  workflow_dispatch:
+
+jobs:
+  build-and-push-arm64:
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        python: ["3.11"]
+    env:
+      # secrets can be set in settings/secrets on github
+      DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: setup QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: setup docker buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver: docker-container
+      - name: expose github runtime for buildx
+        uses: crazy-max/ghaction-github-runtime@v3
+      - name: show system environs
+        run: ./ci/helpers/show_system_versions.bash
+      - name: login to Dockerhub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set deployment variables
+        run: |
+          if [ "${GITHUB_REF}" == "refs/heads/master" ]; then
+            echo "TAG_PREFIX=master-github" >> $GITHUB_ENV
+          elif [[ "${GITHUB_REF}" == refs/heads/hotfix_v* ]]; then
+            echo "TAG_PREFIX=hotfix-github" >> $GITHUB_ENV
+          elif [[ "${GITHUB_REF}" == refs/heads/hotfix_staging_* ]]; then
+            echo "TAG_PREFIX=hotfix-staging-github" >> $GITHUB_ENV
+          fi
+      - name: build & push images for latest tag
+        run: |
+          export DOCKER_IMAGE_TAG="$TAG_PREFIX-latest-arm64"
+          export DOCKER_TARGET_PLATFORMS=linux/arm64
+          make build push=true
+      - name: build & push images for specific tag
+        run: |
+          export DOCKER_IMAGE_TAG=$(exec ci/helpers/build_docker_image_tag.bash)-arm64
+          export DOCKER_TARGET_PLATFORMS=linux/arm64
+          make build push=true
+      - name: fuse images in the registry for latest tag
```

Comment:
what does "fuse" mean here? you **override** `latest` with the arm64 image?

Reply:
the workflow is currently as such and I will detail it more once I can check it works:
- first create the usual amd64 image, and run all the tests as usual
- once this goes through and in master,
- the arm64 images will be built (sadly for now with possibly slightly different libs as the build for arm64 happens at a different timepoint)
- the arm64 build is pushed to dockerhub,
- the arm64 and amd64 images are then "fused" as the registry is able to provide both architectures at the same time